File: app/src/main/java/in/woloo/www/dashboard/ui/invite_friend/InviteFriendViewModel.java
package in.woloo.www.dashboard.ui.invite_friend;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class InviteFriendViewModel extends ViewModel {

    private MutableLiveData<String> mText;

    public InviteFriendViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is gallery fragment");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/java/in/woloo/www/dashboard/ui/invite_friend/InviteFriendViewModel.kt
package `in`.woloo.www.dashboard.ui.invite_friend

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class InviteFriendViewModel : ViewModel() {
    private val mText =
        MutableLiveData<String>()

    init {
        mText.value = "This is gallery fragment"
    }

    val text: LiveData<String>
        get() = mText
}